src/components/MultiCombobox/MultiCombobox.tsx

```diff
@@ -298,6 +309,23 @@ function MultiCombobox<Item>({
                     ))}
                   </Flex>
                 )}
+                {canClearAllAfter && value.length >= canClearAllAfter && (
```

Comment:
`canClearAllAfter` will always be defined

```suggestion
                {value.length >= canClearAllAfter && (
```